specs/spec-2-election-ui.md

```diff
@@ -0,0 +1,200 @@
+
+Mechanisms of Nomination are out of scope as are mechanisms of assigning congress seats. We do need a way to clearly communicate who has been elected to congress seats.
+
+## Use Cases
+
+<!-- Who is using this and what does it do? Leave "how" to the design/tech specification sections. If the solution doesn't help these people achieve their goal then the project isn't successful, even if it fills all of the specified technical requirements. -->
+<!-- List the different groups of people that will use this widget and what each one can specifically do. If more clarity is needed you can include an actor/action matrix. List all users and their types. List the actions each collective will take individually. User profile description. Describe which action is done by which actors. Feel free to use a table format or provide your own graphics or bundle the description in the Actions section. A "swimlane process chart" often works well here. -->
+
+Actions are broken down by actors:
+
+- Voter
+- Candidate
+- Watchdog (no IAH credentials)
+
+These user stories illustrate the needs and desires of the actors rather than perscribing how the actors will achieve their goals.
+
+### Voter
+
+#### Outside of Election UI but still in BOS
+
+- I want to be notified when the election opens and notified when the election is about to close directly on BOS [notification widget].
+- I want to know about upcoming elections and their dates. 
+
+#### Viewing a Live Election
+
+- I want to see all the candidates for each congressional body, so I know my options.
+- I want to see current voting results, so I know everything is happening on chain.
+- I want to see how many candidates I can vote for per congressional body, so I know how to select candidates.
+- I want to see my past votes casted for each congressional body, so I know who I voted for.
+- I want to understand the time frame I'm looking at defined both in terms of Congressional # and year.
+- I want to see the results of an election after it was over with transaction history, so there is accountability after the election.
+
+#### Voting in a Live Election
+
+- I want a cart of candidates, so I can organize who I am voting for before I cast a vote on chain.
+- I want to be able to vote on each Congressional body one at a time so I don’t have to make the entire decision at once.
+- I want to be able to toggle between Congressional bodies so I can more easily choose candidates and my vote status.
+- I want to see how many votes I have cast and have left for each congressional body, to better allocate my votes and understand the quadratic voting process.
+- I want to learn about quadratic voting because it's new to me.
+- I want to easily increment and decrement the votes I've assigned to each canidate. 
```

Comment:
```suggestion
- I want to easily increment and decrement the votes I've assigned to each canidate, and see the global voting credit change (based on quadratic voting).
```

Reply:
let's keep QV for the moment, and we will update it later once we have a decision.